CHANGES.md

```diff
@@ -89,6 +89,10 @@ OpenSSL 3.4
 
    *Craig Lorentzen*
 
+ * ECC groups may now customize their initialization to save CPU.
```

Comment:
Could you please expand this a little? Perhaps: `... initialization to save CPU by using precomputed values. This is used to speed up P-256 curve implementation.`

Reply:
Pushed another fixup that should tweak it